pallets/loans/src/lib.rs

```diff
@@ -89,6 +89,9 @@ pub mod pallet {
     pub trait Config: frame_system::Config {
         type Event: From<Event<Self>> + IsType<<Self as frame_system::Config>::Event>;
 
+        /// Borrow rate used to calculate polynomial rate models.
+        type BorrowRate: Get<Rate>;
```

Comment:
BorrowRate has no usage in this PR.

Reply:
OK. Taking into considerations the feedback, this PR now does not depend on #231 anymore.